File: GreenHauseMonitor/Respones/QuestResponse/QuestResponse.cs
using Cmms.DtoModels;
using System;
using System.Collections.Generic;
using static Cmms.Domain.Dictionary.QuestStateEnum;

namespace Cmms.Respones.QuestResponse
{
    public class QuestResponse
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime DeadLineDataTime { get; set; }
        public int Priority { get; set; }
        public QuestState QuestState { get; set; }
        public Guid QuestTypeId { get; set; }

        IEnumerable<QuestToUserDto> QuestToUsers { get; set; }
        IEnumerable<QuestToEquipmentDto> QuestToEquipments { get; set; }
    }
}

File: cmms.client-react/src/models/Quest/QuestResponse.ts
import { QuestToEquipmentDto } from "./QuestToEquipmentDto"
import { QuestToUserDto } from "./QuestToUserDto"

export interface QuestResponse {
    id: string,
    name: string,
    description: string,
    deadLineDataTime: string,
    priority: number
    questState: number,
    questTypeId: string,

    questToUsers: QuestToUserDto[],
    questToEquipments: QuestToEquipmentDto[]
}